src/main/java/mapWithNameAndPassword/CustomerNameAndPass.java

```diff
@@ -0,0 +1,34 @@
+package mapWithNameAndPassword;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Temporary class for store customers names and passwords
+ */
+public class CustomerNameAndPass {
+
+    public static Map<String, String> map;
+
+    /*
+    This method creates every time new Map, but it is not right.
+    We need to save all entered customers names and passwords.
+    If we create HashMap such as this class field - method doPost from
+    RigisterServlet class will not work.
+     */
+    public void customersNamesAndPasswords(String name, String password) {
```

Comment:
what does this method?

Reply:
I use it in RegisterServlet class to add new Customer in Map